packages/@aws-cdk/aws-rds/test/integ.instance.lit.expected.json

```diff
@@ -645,6 +645,7 @@
         ],
         "EnablePerformanceInsights": true,
         "Engine": "oracle-se1",
+        "EngineVersion": "11.2",
```

Comment:
Does this deploy? I'm not sure you can specify only part of the engine version string here.